cppcache/integration/test/FunctionExecutionTest.cpp

```diff
@@ -145,20 +148,98 @@ TEST(FunctionExecutionTest,
       .jar(getFrameworkString(FrameworkVariable::JavaObjectJarPath))
       .execute();
 
-  auto cache = CacheFactory().set("log-level", "none").create();
-  auto pool = cache.getPoolManager()
-                  .createFactory()
-                  .addLocator("localhost", 10334)
-                  .create("pool");
+  auto cache = cluster.createCache();
+  //  CacheFactory().set("log-level", "none").create();
```

Comment:
just a heads up for this also 🙂

Reply:
Actually, this test (FunctionReturnsObjectWhichCantBeDeserializedOnServer) is disabled because it had a few problems. We created another ticket for it so I just disabled it. But doesn't hurt to delete the commented out piece.

Reply:
Yeah, fixing this function is PR #506 :)

Reply:
ah ok, glad this is just a case of me not keeping up on the other PRs